account_voucher_no_check_default/account_voucher.py

```diff
@@ -68,7 +68,10 @@ def move_include(self, cr, uid, values, val_exclude, context=None):
 
     def recompute_moves(self, cr, uid, ids, partner_id, journal_id, amount, currency_id,
                         ttype, date, values, move_exclude, context=None):
-
+			
+        if context is None:
+            context = {}
+        context = dict(context)
```

Comment:
@urbandesignec 
One line is better than three
this can be accomplished with

`context = dict(context or {})`